index.html

```diff
@@ -162,86 +159,78 @@ <h6 class="title is-5">
-                  </div>
-                  <div class="dropdown-item">
-                    <label class="checkbox">
-                      <input type="checkbox"
-                              name="filters[shouldHaveConcentrator]"
-                              data-filter-name="shouldHaveConcentrator"
-                              value="1"
-                              checked>
-                      Oxygen Concentrators
-                    </label>
-                  </div>
+              <div class="oxygen-filters">
+                <div class="dropdown-item">
+                  <label class="checkbox">
+                    <input type="checkbox" name="filters[shouldHaveRefilling]" data-filter-name="shouldHaveRefilling"
+                      value="1" checked>
+                    Oxygen Refilling
+                  </label>
                 </div>
-
-                <hr class="dropdown-divider">
-
-                <div class="meals-filters">
-                  <div class="dropdown-item">
-                    <label class="checkbox">
-                      <input type="checkbox"
-                              name="filters[shouldHaveFree]"
-                              data-filter-name="shouldHaveFree"
-                              value="1"
-                              checked>
-                      Free Meals
-                    </label>
-                  </div>
+                <div class="dropdown-item">
+                  <label class="checkbox">
+                    <input type="checkbox" name="filters[shouldHaveCylinders]" data-filter-name="shouldHaveCylinders"
+                      value="1" checked>
+                    Oxygen Cylinders
+                  </label>
                 </div>
+                <div class="dropdown-item">
+                  <label class="checkbox">
+                    <input type="checkbox" name="filters[shouldHaveCans]" data-filter-name="shouldHaveCans" value="1"
+                      checked>
+                    Oxygen Cans
+                  </label>
+                </div>
+                <div class="dropdown-item">
+                  <label class="checkbox">
+                    <input type="checkbox" name="filters[shouldHaveConcentrator]"
+                      data-filter-name="shouldHaveConcentrator" value="1" checked>
+                    Oxygen Concentrators
+                  </label>
+                </div>
+              </div>
 
+              <hr class="dropdown-divider">
+
+              <div class="meals-filters">
+                <div class="dropdown-item">
+                  <label class="checkbox">
+                    <input type="checkbox" name="filters[shouldHaveFree]" data-filter-name="shouldHaveFree" value="1"
+                      checked>
+                    Free Meals
+                  </label>
                 </div>
               </div>
+
             </div>
           </div>
         </div>
       </div>
+
+      <div class="select">
```

Comment:
Simply move this into the two divs above so that it is a sibling of `<div class="dropdown">` and it will appear next to filter dropdown.